src/app/(proper_react)/(redesign)/(public)/how-it-works/components/FooterSection.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Button } from "../../../../../components/client/Button";
+import styles from "../HowItWorksView.module.scss";
+import { ExtendedReactLocalization } from "../../../../../functions/l10n";
+
+export const FooterSection = ({
+  l10n,
+}: {
+  l10n: ExtendedReactLocalization;
+}) => {
+  return (
+    <footer className={styles.footerSection}>
+      <h2 className={styles.footerSectionCTA}>
+        {l10n.getString("footer-title")}
+      </h2>
+      <div className={styles.footerSectionCTAInput}>
+        <input
+          placeholder={l10n.getString("footer-input-placeholder")}
+          type="email"
+        />
+        <Button variant="primary">{l10n.getString("footer-cta-button")}</Button>
```

Comment:
Does the `input` and `Button` need to be wrapped in a form? Or is there some sneaky submit handler somewhere that is doing something with that email input?

Reply:
At the moment the input does not seem to work. We might be able to reuse the one from the landing page, which is the `SignUpForm` component?

Reply:
At the moment the input does not seem to work. We might be able to reuse the one from the landing page, which is the `SignUpForm` component?

Reply:
At the moment the input does not seem to work. We might be able to reuse the one from the landing page, which is the `SignUpForm` component?

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/a9d4b2ca01034d57496df4da39000af276b6cfaa